src/termio/shell_integration.zig

```diff
@@ -76,13 +77,15 @@ pub fn setup(
     return result;
 }
 
-fn setupShell(
-    alloc_arena: Allocator,
-    resource_dir: []const u8,
-    command: config.Command,
-    env: *EnvMap,
-    exe: []const u8,
-) !?ShellIntegration {
+fn setupShell(alloc_arena: Allocator, resource_dir: []const u8, command: config.Command, env: *EnvMap, exe: []const u8, features: config.ShellIntegrationFeatures) !?ShellIntegration {
+    if (std.mem.eql(u8, "nu", exe)) {
```

Comment:
These cases were listed alphabetically before. Let's retain that ordering and move this between "fish" and "zsh".